src/basho_bench_stats_writer_csv.erl

```diff
@@ -88,6 +97,46 @@ report_error({_SummaryFile, ErrorsFile},
                io_lib:format("\"~w\",\"~w\"\n",
                              [Key, Count])).
 
+report_global_stats({Op,_}, Stats, Errors, Units) ->
+    %% Build up JSON structure representing statistices collected in folsom
+    P = proplists:get_value(percentile, Stats),
+    JsonElements0 = lists:foldl(fun(K, Acc) ->
+        case K of
+            %% Unpack percentiles list
+            percentile ->
+                [
+                    % we skip p50 since we have the median
+                    {p75, proplists:get_value(75, P)},
+                    {p90, proplists:get_value(90, P)},
+                    {p95, proplists:get_value(95, P)},
+                    {p99, proplists:get_value(99, P)},
+                    {p999, proplists:get_value(999, P)}
+                ] ++ Acc;
+
+            %% We ignore the histogram as it will not be used in the analysis
+            histogram -> Acc;
+
+            %% Rename arithmetic_mean to mean {<key>, <value>}
+            arithmetic_mean ->
+                [{mean, proplists:get_value(K, Stats)} | Acc];
+
+            %% All other metrics get turned into {<key>, <value>}
+            _ ->
+                [{K, proplists:get_value(K, Stats)} | Acc]
+        end
+    end, [], proplists:get_keys(Stats)),
+
+    %% insert correct units count
+    JsonElements1 = lists:keyreplace(n, 1, JsonElements0, {'_n', Units}),
```

Comment:
not a typo, it's been something that we adopted at some point, a long time ago. 
It's been bugging me, but no more, i've changed it internally on nimbus. so I will be changing it to 'n'